app/assets/stylesheets/users.scss

```diff
@@ -0,0 +1,61 @@
+
+/* user index table */
+div.panel.panel-default {
+	display: table;
+}
+
+.panel-default > .panel-heading {
+	background-color: #819bd2;
+	color: white;
+	border-bottom: 1px solid #627AAD;
+}
+
+div.panel {
+	margin: 5px;
+	width:	400px;
+}
+
+/* x button */
+
+.x-button {
```

Comment:
It is better to give more meaningful names. This could be close-button or delete-button